src/tv2-common/migrations/index.ts

```diff
@@ -173,6 +174,74 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string, fromValue: string, targetTable: string) {
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupName.${fromValue}.ToTable.${targetTable}`,
```

Comment:
Be nice if you were consistent with your casing. You are using both camelCasing and PascalCasing here

Reply:
Fixed in the newest commit!